torch/testing/_internal/distributed/rpc/jit/rpc_test.py

```diff
@@ -101,10 +111,42 @@ def rref_script_annotation(rref_var):
     return rref_python_annotation(rref_var).to_here()
 
 
+class LocalRRefTest(RpcAgentTestFixture):
+    @dist_init
+    def test_create_local_ivalue_rref(self):
+        if self.rank != 0:
+            return
+
+        # Create a local RRef<MyScriptClass>.
+        rref_script_class = rpc.RRef(MyScriptClass(10))
+        ret = rref_script_class.to_here().get_value()
+        self.assertEqual(ret, 10)
+
+        # Create a local RRef<MyModuleInterface>.
+        rref_script_module = rpc.RRef(MyScriptModule(3))
+        ret = rref_script_module.to_here().forward()
+        self.assertEqual(ret, torch.ones(3))
+
+    @dist_init
+    def test_return_local_ivalue_rref_in_py_and_use_in_script(self):
+        if self.rank != 0:
+            return
+
+        dst_worker_name = "worker{}".format((self.rank + 1) % self.world_size)
+
+        # Remote side creates and returns a local RRef<MyScriptModuleInterface> in Python.
+        rref = rpc.rpc_sync(dst_worker_name, owner_create_rref_my_script_module, args=(3,))
+        # Remote side uses the RRef in Script.
+        ret = rpc.rpc_sync(
+            rref.owner(), script_run_forward_rref_my_script_module, args=(rref,)
```

Comment:
Will the error be propagated back properly if there is any?

Reply:
Added test for wrong type failure case.